batch/batch/front_end/front_end.py

```diff
@@ -73,7 +73,7 @@
     return web.Response()
 
 
-async def _query_batch_jobs(request, batch_id):
+async def _query_batch_jobs(request, batch_id, include_spec_status=True):
```

Comment:
So I'm going to argue that query_jobs should never return the full spec status.  If the client wants it, it should loop over the jobs and get the job details for each job.  This might involve changing some tests, yes.  This isn't a huge deal since we LIMIT 50 here, but in general I think it is good to keep the number of additional requests made to satisfy a request constant and small (e.g. 1-3) if possible.  This requires 1-3 for each job.